src/home.js

```diff
@@ -0,0 +1,15 @@
+// const { container } = require("webpack");
+
```

Comment:
- [ ] Kindly remove commented out code. This will keep your repository clean. Always remember you can use git history to go back to the previous version of your code. ⬇️ 
<img width="792" alt="no_inline_comments" src="https://user-images.githubusercontent.com/32361963/111418439-c92b4680-86f8-11eb-985e-c5e27337e466.png">